ql.mm

```diff
@@ -10305,84 +9783,82 @@ Kalmbach axioms (soundness proofs) that require WOML
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.2e2 $p |- ( ( a ->1 b ) ^ b ) = ( ( a ^ b ) v ( a ' ^ b ) ) $=
     ( u1lemab ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.11 of [MegPav2000] p. 23.  This is the first part of the
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.3le1 $p |- ( a ' ->1 b ) ' =< a ' $=
     ( u1lem9a ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.11 of [MegPav2000] p. 23.  This is the second part of the
-     equation.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     equation.  (Contributed by Roy F. Longton, 1-Jul-2005.)  (Revised by Roy
+     F. Longton, 3-Jul-2005.) $)
   lem4.6.3le2 $p |- a ' =< ( a ->1 b ) $=
     ( u1lem9b ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.12 of [MegPav2000] p. 23.  (Contributed by Roy F. Longton,
-     3-Jul-05.) $)
+     1-Jul-2005.)  (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.4 $p |- ( ( a ->1 b ) ->1 b ) = ( a ' ->1 b ) $=
     ( u1lem12 ) ABC $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.13 of [MegPav2000] p. 23.  (Contributed by Roy F. Longton,
-     3-Jul-05.) $)
+     1-Jul-2005.)  (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.5 $p |- ( ( a ->1 b ) ' ->1 b ) = ( a ->1 b ) $=
     ( wi1 wn u1lemn1b ax-r1 ) ABCZGDBCABEF $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 1.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 1.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j1 $p |- ( ( a ->0 b ) v ( a ->1 b ) ) = ( a ->0 b ) $=
     ( wn wo wa wi0 wi1 leid lear lelor lel2or leo lebi df-i0 df-i1 2or 3tr1 ) A
     CZBDZRABEZDZDZSABFZABGZDUCUBSSSUASHTBRABIJKSUALMUCSUDUAABNZABOPUEQ $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 2.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 2.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i0j2 $p |- ( ( a ->0 b ) v ( a ->2 b ) ) = ( a ->0 b ) $=
     ( wn wo wa wi0 wi2 leid leor leao1 lel2or leo lebi df-i0 df-i2 2or 3tr1 ) A
     CZBDZBRBCZEZDZDZSABFZABGZDUDUCSSSUBSHBSUABRIRTBJKKSUBLMUDSUEUBABNZABOPUFQ
     $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 0, and j is set to 3.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 0, and j is set to 3.  (Contributed by Roy F. Longton, 1-Jul-2005.)
```

Comment:
The Contributed line should be 3-Jul-2005 and there should be no Revised by. I realize that the original has both dates but there's no point in having two dates two days apart.

(uh oh, how many of these are there?)